PackageViewModel/PackageViewModel.csproj

```diff
@@ -14,6 +14,21 @@
     <ProjectReference Include="..\Core\Core.csproj" />
     <Reference Include="System.ComponentModel.Composition" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
@campersau: These were added automatically when I opened the solution. Should I remove them? If I do, they'll get added the next time I open the sln anyway 🤓

Reply:
Strange. For me they got only added the very first time I opened the solution. After I removed these and reopend the solution they weren't added again.
But yes it would good if you could remove them. Or I can do it once the main UI features are in.

Reply:
Ok, I can try. I'll update you if I can't get them removed permanently.